src/libcollections/vec_deque.rs

```diff
@@ -949,8 +949,8 @@ impl<T> VecDeque<T> {
     /// buf.insert(1,11);
     /// assert_eq!(Some(&11), buf.get(1));
     /// ```
-    pub fn insert(&mut self, i: usize, t: T) {
-        assert!(i <= self.len(), "index out of bounds");
+    pub fn insert(&mut self, index: usize, value: T) {
+        assert!(index <= self.len(), "index out of bounds");
```

Comment:
Note that several inner impl docs refer to `i` explicitly (this fn is an example).